src/Adafruit_BNO08x.cpp

```diff
@@ -288,7 +312,7 @@ static int i2chal_read(sh2_Hal_t *self, uint8_t *pBuffer, unsigned len,
                        uint32_t *t_us) {
   // Serial.println("I2C HAL read");
 
-  uint8_t *pBufferOrig = pBuffer;
+  // uint8_t *pBufferOrig = pBuffer;
```

Comment:
Just delete this instead?

Reply:
this one is potentially used later in commented out debug prints, so needs to remain as a commented out line.